src/nim_status_client.nim

```diff
@@ -47,7 +47,7 @@ proc mainProc() =
   enableHDPI()
   initializeOpenGL()
 
-  let app = newQApplication("Status Desktop")
+  let app = newQGuiApplication()
```

Comment:
Since the `Status Desktop` argument was removed,  wouldnt we have to call `setApplicationDisplayName` to set the title?

Reply:
If you were diving deeper and saw where what "Status Desktop" argument went... it went to argv[0] that supposed to represent the executable filepath. `"Status Desktop"` string is not an executable filepath and therefore it never was the "application display name".
What you are looking for is there already, in main.qml:46
`Qt.application.name = qsTrId("status-desktop")`

Reply:
First - why its should be translated? and how executable name have to do with display name?

Reply:
https://stackoverflow.com/a/38371112

Reply:
Sorry but i don't see any answers on any questions in this thread.

Reply:
> and how executable name have to do with display name?

> From V5, these will fall back to the executable name if the property is not set.

As for the first question: I quoted existing code from main.qml. I dont know why someone decided to translate it.